pkg/inspect/getter.go

```diff
@@ -43,6 +43,19 @@ func GetObjectPropString(ctx context.Context, obj interface{}, prop string) (str
 	return "", fmt.Errorf("%s is not a string, found %s", prop, c.Kind().String())
 }
 
+func GetObjectArray(obj interface{}, prop string) ([]interface{}, error) {
+	v, err := getObjectProp(obj, prop)
+	if err != nil {
+		return nil, err
+	}
+	var result []interface{}
+	for i := 0; i < v.Len(); i++ {
```

Comment:
That will panic with the wrong params. Test the `Kind()` and if slice, array, then proceed.